More details create credit and create tax items endpoints (#166)
killbill/slate: source/includes/_invoice.md

### Create tax items

> Normally, tax items are added to an invoice by a plugin, which intercepts the invoice during its creation and adds the required tax items on the fly. However, sometimes you may want to add tax items directly. This API adds a tax item to an account, which will result in the creation of a new invoice. Also, it is worth noting, that the tax items are not linked to any existing `invoiceItem`s. Consequently, if you pass certain fields like `linkedInvoiceItemId`, `subscriptionID` while creating the tax item, they will not reflect in the database and the same will not be visible in subsequent Get requests. 

Reviewer: ```suggestion
Normally, tax items are added to an invoice by a plugin, which intercepts the invoice during its creation and adds the required tax items on the fly. However, sometimes you may want to add tax items directly. This API adds a tax item to an account, which will result in the creation of a new invoice. Also, it is worth noting, that the tax items are not linked to any existing `invoiceItem`s. Consequently, specifying fields like `linkedInvoiceItemId`, `subscriptionId` while creating the tax item has no effect. These fields do not get saved on the database and will not be returned in subsequent `GET` requests.
```
